app/views/layouts/application.html.erb

```diff
@@ -3,6 +3,10 @@
   <head>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Rails Project</title>
+
+    <%# Boxicons %>
+    <link href='https://unpkg.com/boxicons@2.1.1/css/boxicons.min.css' rel='stylesheet'>
```

Comment:
Why don't we install this library to this app?

Reply:
installed box icons library